include/deal.II/multigrid/mg_constrained_dofs.h

```diff
@@ -50,6 +50,15 @@ class MGConstrainedDoFs : public Subscriptor
    * edges of a refinement level are treated corrected but leaves degrees of
    * freedom at the boundary of the domain untouched assuming that no
    * Dirichlet boundary conditions for them exist.
+   *
+   * Furthermore, this call sets up a ConstraintMatrix on each level that
```

Comment:
I think it's also a correctness issue: if the user declared to the triangulation that two boundaries are periodic, we ought to honor that.